phpcs.xml

```diff
@@ -17,6 +17,5 @@
 	<exclude-pattern>vendor/</exclude-pattern>
 
 	<!-- WordPress Coding Standards -->
-	<rule ref="WordPress-Core"></rule>
-	<rule ref="WordPress-Extra"></rule>
+	<rule ref="WordPress-Extra" />
```

Comment:
`WordPress-Extra` includes `WordPress-Core`, so there's no need to define both.

See https://github.com/WordPress/WordPress-Coding-Standards?tab=readme-ov-file#rulesets